package-lock.json

```diff
@@ -177,6 +177,32 @@
       "resolved": "https://registry.npmjs.org/camelcase/-/camelcase-3.0.0.tgz",
       "integrity": "sha1-MvxLn82vhF/N9+c7uXysImHwqwo="
     },
+    "chart.js": {
```

Comment:
There's no need for a package-lock.json file, could you please remove this? Also I'd recommend using yarn instead of npm so you don't have this problem!